packages/calcite-components/src/components/accordion-item/accordion-item.scss

```diff
@@ -1,195 +1,207 @@
-%icon-position {
-  /* icon rotation variables */
-  --calcite-accordion-item-icon-rotation: calc(theme("rotate.90") * -1);
-  --calcite-accordion-item-active-icon-rotation: theme("rotate.0");
-  --calcite-accordion-item-icon-rotation-rtl: theme("rotate.90");
-  --calcite-accordion-item-active-icon-rotation-rtl: theme("rotate.0");
-}
-
-// icon position variants for child
-.icon-position--start {
-  @extend %icon-position;
-  --calcite-accordion-item-flex-direction: row-reverse;
-  --calcite-accordion-item-icon-spacing-start: 0;
-  --calcite-accordion-item-icon-spacing-end: var(--calcite-accordion-icon-margin);
-}
-
-.icon-position--end {
-  @extend %icon-position;
-  --calcite-accordion-item-flex-direction: row;
-  --calcite-accordion-item-icon-spacing-start: var(--calcite-accordion-icon-margin);
-  --calcite-accordion-item-icon-spacing-end: 0;
-}
-
-.icon-position--end:not(.icon-type--plus-minus) {
-  --calcite-accordion-item-icon-rotation: theme("rotate.0");
-  --calcite-accordion-item-active-icon-rotation: theme("rotate.180");
-  --calcite-accordion-item-icon-rotation-rtl: theme("rotate.0");
-  --calcite-accordion-item-active-icon-rotation-rtl: calc(theme("rotate.180") * -1);
-}
+/**
+ * CSS Custom Properties
+ *
+ * @prop --calcite-accordion-item-background: The background color of the accordion item.
+ * @prop --calcite-accordion-item-border-color: The border color of the accordion item.
+ * @prop --calcite-accordion-item-outline-color-focus: The outline color of an accordion item when it is in focus;
+ * @prop --calcite-accordion-item-outline-color: The outline color of an accordion item;
+ * @prop --calcite-accordion-item-padding: Deprecated. Please use --calcite-accordion-spacing instead.
+ * @prop --calcite-accordion-item-secondary-text-color-selected: The text color of the secondary text in the accordion item when selected.
+ * @prop --calcite-accordion-item-text-color-selected: The text color of the accordion item when it is selected.
+ * @prop --calcite-accordion-item-text-color: The text color of the accordion item.
```

Comment:
Should this be `heading`? Users can already control the color of the content they slot - but this doesn't seem to convey this would change the `heading` rendered via prop.